### PR TITLE
Longest Consecutive subsequence in python 

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes 

### Description

Adding the file in NeoAlgo\Python\other\Longest_Consecutive_Subsequence.py 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6261

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by soumikbaithalu** [comment]
> Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.
 
Yes, the file is formatted as the instruction is said in a proper code format. @ankitaggarwal23

File: Python/other/Longest_Consecutive_Subsequence.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+numbers form the longest consecutive
+subsquence.
+
+Example 2:
+Input: {1,9,3,10,4,20,2}
+Output: 4
+
+Explanation:
+1, 2, 3, 4 is the longest
+consecutive subsequence.
```
Comment (ankitaggarwal23) [inline Python/other/Longest_Consecutive_Subsequence.py]: REMOVE THIS PART

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/other/Longest_Consecutive_Subsequence.py (hunk 1 above)
Comment (soumikbaithalu) [reply Python/other/Longest_Consecutive_Subsequence.py]: But sir this will help to under stand beginners user

Comment (soumikbaithalu) [reply Python/other/Longest_Consecutive_Subsequence.py]: @ankitaggarwal23 is this okay

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by soumikbaithalu** [comment]
@ankitaggarwal23 this medium level atleast level 2 should be there

**Comment by soumikbaithalu** [comment]
Screenshot_20210422_183547  @ankitaggarwal23 if you do not believed me pleases check GFG and leetcode here

**Comment by soumikbaithalu** [comment]
..

**Comment by anushkrishnav** [comment]
Sorry, but we do not use the difficulty rating by GFG or Leetcode rather we use the criteria set by the GSSoC admins a few weeks back.

**Pull request merged**